sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample11_RecognizeIdDocuments.md

```diff
@@ -0,0 +1,210 @@
+# Recognize Id Documents
```

Comment:
Just pointing out that we may change `Id` => `ID` in a lot of places in this file depending on the naming approach we take.

Reply:
changed to ID